### PR TITLE
Bug/new contract fix

### LINKED ISSUE #3822: New contract handler is not called after contract deployment
Problem

var simplestorageContract = web3.eth.contract([{"constant":false,"inputs":[{"name":"x","type":"uint256"}],"name":"set","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"get","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"}]);
var simplestorage = simplestorageContract.new(
   {
     from: web3.eth.accounts[0], 
     data: '0x6060604052341561000f57600080fd5b60d38061001d6000396000f3006060604052600436106049576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff16806360fe47b114604e5780636d4ce63c14606e575b600080fd5b3415605857600080fd5b606c60048080359060200190919050506094565b005b3415607857600080fd5b607e609e565b6040518082815260200191505060405180910390f35b8060008190555050565b600080549050905600a165627a7a7230582087e84161fdd2e7143a7ef1adc914230c9751de6d8abb28fce70837c6d45694c90029', 
     gas: '100000'
   }, function (e, contract){
    console.log(e, contract);
    if (typeof contract.address !== 'undefined') {
         console.log('Contract mined! address: ' + contract.address + ' transactionHash: ' + contract.transactionHash);
    }
 })
Implementation

function (e, contract){ should be called with contract.address after contract will be mined

status go related issue https://github.com/status-im/status-go/issues/784

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes contracts creation

### CODE REVIEW

**Comment by rasom** [comment]
@flexsurfer #3826 was merged

**Review by rasom — APPROVED** [review]

File: src/status_im/ui/screens/wallet/send/views.cljs [hunk 1]
```diff
@@ -84,7 +84,10 @@
 ;; "Sign Later" and "Sign Transaction >" buttons
 (defn- sign-buttons [amount-error to amount sufficient-funds? sign-later-handler]
   (let [sign-enabled? (sign-enabled? amount-error to amount)
-        immediate-sign-enabled? (and sign-enabled? sufficient-funds?)]
+        immediate-sign-enabled? (and sign-enabled? sufficient-funds?)]))
+(defn- sign-buttons [amount-error to amount sufficient-funds? sign-later-handler modal?]
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/send/views.cljs]: Twice sign-buttons ?

File: src/status_im/ui/screens/wallet/send/views.cljs [hunk 2]
```diff
@@ -227,7 +230,7 @@
         [sign-buttons amount-error to amount sufficient-funds? (if modal? (if from-chat?
                                                                             #(sign-later-popup true)
                                                                             #(re-frame/dispatch [:navigate-back]))
-                                                                          #(sign-later-popup false))])
+                                                                   #(sign-later-popup false)) modal? #(sign-later-popup false)])
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/send/views.cljs]: Same here?

File: src/status_im/ui/screens/wallet/send/views.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/send/views.cljs]: yes thanks, rebase

**Comment by flexsurfer** [comment]
@jeluard fixed

**Review by jeluard — APPROVED** [review]

**Comment by flexsurfer** [comment]
for QA: you can use https://status-im.github.io/dapp/, if don't know Dapp which creates new contract

<blockquote><div><strong><a href="https://status-im.github.io/dapp/">DAPP</a></strong></div></blockquote>

**Pull request merged**